### PR TITLE
Make sure the sessionId changes when the app is opened again

### PR DESCRIPTION
**Pull request by gnardini**
### Description

This PR: https://github.com/valora-inc/wallet/pull/1741 made it so the `sessionId` we send of analytics was always the same value.

We now generate a new one based on the device id + date again.

### Other changes

N/A

### Tested

Manually

### How others should test

No need

### Related issues

- Fixes #

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/analytics/ValoraAnalytics.ts [hunk 1]
```diff
@@ -72,14 +72,13 @@ class ValoraAnalytics {
...
-
       await Analytics.setup(SEGMENT_API_KEY, SEGMENT_OPTIONS)
 
       try {
         const deviceInfo = await getDeviceInfo()
         this.deviceInfo = deviceInfo
-        this.sessionId = sha256(
-          Buffer.from('0x' + deviceInfo.UniqueID.split('-').join('') + String(Date.now()), 'hex')
+        this.sessionId = sha256FromString(
+          '0x' + deviceInfo.UniqueID.split('-').join('') + String(Date.now())
```
[the remark this answers is not in the document]
Comment (gnardini) [reply packages/mobile/src/analytics/ValoraAnalytics.ts:81]: Hmm, I didn't investigate that far, I just noticed that the `sessionId` had always the same value but for that to happen then yes, it has to I think.

[the remark this answers is not in the document]
Comment (silasbw) [reply packages/mobile/src/analytics/ValoraAnalytics.ts:81]: Did we consider using an UUID for this?  E.g., using https://www.npmjs.com/package/uuid

Or is it important that the random session ID is a function of the device ID?

[the remark this answers is not in the document]
Comment (jeanregisser) [reply packages/mobile/src/analytics/ValoraAnalytics.ts:81]: It could be just a UUID indeed.

[the remark this answers is not in the document]
Comment (jeanregisser) [reply packages/mobile/src/analytics/ValoraAnalytics.ts:81]: Also, if I remember correctly, Segment already has an internal session id. We should check how it behaves under the hood and possibly deprecate our custom field.
The problem with our current `sessionId` is that it doesn't change until the next time the app is launched, which could be days depending on the backgrounding behavior. Which doesn't represent actual user sessions.

**Comment by ValoraQA** [comment]
Hey @MuckT since "no need" is mentioned in how others should test section we are skipping this PR

CC : @gnardini

**Pull request merged**